app/components/primer/alpha/tool_tip.ts

```diff
@@ -110,15 +100,6 @@ class ToolTipElement extends HTMLElement {
         }
       }
 
-      :host:after{
-        position: absolute;
-        display: block;
-        right: 0;
-        left: 0;
-        height: 12px;
-        content: "";
-      }
```

Comment:
I think we need this too keep the tooltip open when cursor leaves the trigger to hover the tooltip text.